src/main/java/pl/javastart/task/Drink.java

```diff
@@ -0,0 +1,29 @@
+package pl.javastart.task;
+
+import java.sql.SQLOutput;
+
+public class Drink {
+    public static void main(String[] args) {
+        String drink = "Mojito";
+        Boolean alcohol = true;
+        double price = 13.0;
```

Comment:
to powinny być pola (czyli definiowane na poziomie klasy), a nie zmienne (definiowane w środku w metodzie)